invokeai/app/api/routers/images.py

```diff
@@ -22,6 +22,101 @@
 # images are immutable; set a high max-age
 IMAGE_MAX_AGE = 31536000
 
+@images_router.post(
+    "/upload_multiple",
+    operation_id="upload_multiple_images",
+    responses={
+        201: {"description": "The images are being prepared for upload"},
+        415: {"description": "Images upload failed"},
+    },
+    status_code=201,  
+    response_model=Dict[str, str],
+)
+async def upload_images(
+    files: List[UploadFile],
+    background_tasks: BackgroundTasks,
+    request: Request,
+    response: Response,
+    image_category: ImageCategory = Query(description="The category of the images"),
+    is_intermediate: bool = Query(description="Whether this is an intermediate images"),
+    board_id: Optional[str] = Query(default=None, description="The board to add this images to, if any"),
+    session_id: Optional[str] = Query(default=None, description="The session ID associated with this upload, if any"),
```

Comment:
`session_id` is no longer used for uploads - this was needed for the canvas at some point in the past. Can omit this. We'll need to clean it up in the single upload route too.